web/backend/src/controllers/authentication.ts

```diff
@@ -0,0 +1,94 @@
+import express from 'express';
+import axios, { AxiosError } from 'axios';
+import { sciper2sess } from '../session';
+import { getUserPermissions, setMapAuthorization } from '../authManager';
+
+export const authenticationRouter = express.Router();
+
+// This is via this endpoint that the client request the tequila key, this key
+// will then be used for redirection on the tequila server
+authenticationRouter.get('/get_teq_key', (req, res) => {
+  const body = `urlaccess=${process.env.FRONT_END_URL}/api/control_key\nservice=Evoting\nrequest=name,firstname,email,uniqueid,allunits`;
```

Comment:
normally we're aiming for an 80 char (soft) limit and 100 (hard) limit.
You might want to split this line, especially as template strings (like this one) do support line breaks within them, and it would be more readable than inlined `\n`

Reply:
hmm.. do you mean like this? 

```javascript
const body = `urlaccess=${process.env.FRONT_END_URL}/api/control_key
                         \nservice=Evoting
                         \nrequest=name,firstname,email,uniqueid,allunits`;

```

Reply:
Isn't the `\n` in there superfluous ? 
i.e.

```javascript
const body = `urlaccess=${process.env.FRONT_END_URL}/api/control_key
service=Evoting
request=name,firstname,email,uniqueid,allunits`;
```

Reply:
@pierluca I tried it like that as well.. but it didn't work. I didn't spent time to know why.

Reply:
The first version with the `\n` has two newlines between every line. Perhaps that changes something?

Reply:
I removed this completely, replacing it with the following: 

```
  axios
    .get(`https://tequila.epfl.ch/cgi-bin/tequila/createrequest`, {
      params: {
        urlaccess: `${process.env.FRONT_END_URL}/api/control_key`,
        service: 'Evoting',
        request: 'name,firstname,email,uniqueid,allunits',
      },
    })
```

Yes, this works :D

Reply:
And this is so much better and cleaner !